### PR TITLE
July trains - error codes and required property

### PR DESCRIPTION
**Pull request by cristobal-buenrostro**
This PR is related to the issues below:

https://github.com/microsoftgraph/microsoft-graph-docs/issues/13701

https://github.com/microsoftgraph/microsoft-graph-docs/issues/13702

### CODE REVIEW

File: api-reference/beta/api/educationoutcome-update.md [hunk 1]
```diff
@@ -51,6 +51,24 @@ All derived outcome types have a regular and a "published" property appropriate
 
 If successful, this method returns a `200 OK` response code and an updated [educationOutcome](../resources/educationoutcome.md) object in the response body.
 
+If using **PointsGradeType** and **Points** are updated to be negative or infinite, a `400` error is returned as below.
```
Comment (isvargasmsft) [inline api-reference/beta/api/educationoutcome-update.md:54]: Isn't this behavior applicable for the POST (create) operation as well? Please confirm and include if applicable.

Comment (isvargasmsft) [inline api-reference/beta/api/educationoutcome-update.md:54]: This is one of the scenarios when we actually can use passive voice, please rephrase for clarity:

If **PointsGradeType** and **Points** are updated to a negative or infinite value, the method returns a `400` error message.

Comment (isvargasmsft) [reply api-reference/beta/api/educationoutcome-update.md:54]: Please apply in v1 as well.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @cristobal-buenrostro, 

Thank you very much for your work in this PR. 
1. Please confirm whether this behavior applies to the POST operation as well. 
2. Please rephrase for clarity.

File: api-reference/beta/api/educationoutcome-update.md (hunk 1 above)
Comment (cristobal-buenrostro) [reply api-reference/beta/api/educationoutcome-update.md]: not applicable

Comment (cristobal-buenrostro) [reply api-reference/beta/api/educationoutcome-update.md]: done, changed as suggested in both

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**